src/Symfony/Component/Form/Extension/Core/Type/UrlType.php

```diff
@@ -47,7 +48,11 @@ public function buildView(FormView $view, FormInterface $form, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'default_protocol' => 'http',
+            'default_protocol' => static function (Options $options) {
+                trigger_deprecation('symfony/form', '6.4', 'Not configuring the "default_protocol" option is deprecated. It will default to "null" in 7.0.');
```

Comment:
```suggestion
                trigger_deprecation('symfony/form', '6.4', 'Not configuring the "default_protocol" option when using the UrlType is deprecated. It will default to "null" in 7.0.');
```